tdrs-frontend/src/components/Modal/Modal.jsx

```diff
@@ -24,18 +24,17 @@ const Modal = ({ title, message, buttons = [], isVisible = false }) => {
         (b) => b.key === selectedButtonKey
       )
 
+      const btnIdxMinOne = Math.max(0, buttons.length - 1)
       if (shiftKey) {
         // go backward
+        const selectedIdxMinOne = Math.max(0, selectedButtonIndex - 1)
```

Comment:
This fixed a bug where we could get an array out of bounds error. We could also escape the modal with tabs when this error occurred.